Assets/Vimeo/Scripts/TestConfigSample.cs

```diff
@@ -17,5 +17,7 @@ public class TestConfigSample
     public const string TEST_PROJECT_FOLDER = "/users/xxx/projects/xxx";
     public const string VERY_BIG_FILE_PATH = "path/to/very/big/file.mp4";
     public const string TEST_IMAGE_PATH = "D:/dev/vimeo-unity-sdk/Assets/Vimeo/Images/vimeo_v_blue.png";
+    public const string TEST_ADAPTIVE_LINK_TO_UNFURL = "some_link_from_the_vimeo_api_to_an_adaptive_video";
```

Comment:
Dont think we should use test constants that are expiring links. It will make it hard for us to test on an ongoing basis. Either we:
- Use a permanent test URL that we know we can unfurl and doesnt expire
- Or do a full integration test

Reply:
I think full integration tests make more sense - added in https://github.com/vimeo/vimeo-unity-sdk/commit/6d66326496b8f4ccc1095287ed28858880e05a61